scripts/build/builders/host.py

```diff
@@ -569,6 +570,13 @@ def __init__(self, root, runner, app: HostApp, board=HostBoard.NATIVE,
             else:
                 self.extra_gn_options.append('chip_terms_and_conditions_required=false')
 
+        if use_otep:
+            self.extra_gn_options.append('chip_system_config_use_openthread_inet_endpoints=true')
+            self.extra_gn_options.append('chip_enable_wifi=false')
```

Comment:
should we make extra checks regarding `enable-wifi` ? or maybe only support the option if the explicit wifi disabling argument is set?